contracts/governance/extensions/GovernorTimelockControl.sol

```diff
@@ -17,6 +17,9 @@ import "../TimelockController.sol";
  * inaccessible.
  *
  * _Available since v4.3._
+ *
+ * WARNING: Do not share Timelock with several proposers.
```

Comment:
We should give a brief explanation of the risks that motivate this recommendation.